ipatests/test_integration/test_sssd.py

```diff
@@ -75,6 +75,14 @@ def install(cls, mh):
         cls.users['child_ad']['name'] = (
             cls.users['child_ad']['name_tmpl'].format(
                 domain=cls.child_ad.domain.name))
+
+        # workaround for https://pagure.io/freeipa/issue/8294
+        # enable AES128 and AES256 encryption on AD
+        cmd = ("ksetup /setenctypeattr {0} AES128-CTS-HMAC-SHA1-96 "
+               "AES256-CTS-HMAC-SHA1-96")
+        cls.child_ad.run_command(cmd.format(cls.ad.domain.name))
+        cls.ad.run_command(cmd.format(cls.child_ad.domain.name))
+
```

Comment:
These steps will probably be needed by other tests using ad domain + child domain, I would put them in the tasks module.
The commit message can also be changed from "Related: http..." to "Fixes: http..."